### DISCUSSION TITLE
Affection log Error

### DISCUSSION THREAD

**Discussion by ZeroKich** [Monika-After-Story/MonikaModDev#8673] [QA - tech support + general]
My affection log is only displaying the affection I had on 1/30 and it's 2/4 at the time of making this post. What's worse is that the affection is lower then the number I had on 1/30. And advice on what to do?

Edit: So I checked a few hours later and it loaded fine. I have no idea what happened.

**Comment by multimokia** [comment]
Are you looking at `afflog.txt` or `afflog.log`?

**Reply by ZeroKich** [reply]
I meant afflog.log
